inst/tests/tests.Rraw

```diff
@@ -15864,6 +15864,22 @@ test(2095, any(grepl('override', capture.output(dcast(DT, a~b, fun.aggregate=len
 # gmean intermediate can overflow integers without warning, #986
 test(2096, data.table(a=c(1L,1L), v=c(2e9L, 2e9L))[, mean(v), a], data.table(a=1L, V1=2e9))
 
+# fifelse, #3753, add argument na for missing values
```

Comment:
since it's quite recent fix, I would include these tests in the old suite

Reply:
where exactly do you want me to move them to?

Reply:
`fifelse` tests are already scattered in non-continuous chunks so I think it is OK as is.